### ISSUE TITLE
UMD modules and moduleId

### ISSUE THREAD

**Issue by kobezzza** [babel/babel#1165]
**index.js**

```js
const myLib = {};
export default myLib;
```

```js
options: {
  modules: 'umd',
  moduleId: 'MyLib'
}
```

->

```js
(function (global, factory) {
  if (typeof define === 'function' && define.amd) {
    define('MyLib', ['exports', 'module'], factory);
  } else if (typeof exports !== 'undefined' && typeof module !== 'undefined') {
    factory(exports, module);
  } else {
    var module = {
      exports: {}
    };
    factory(module.exports, module);
    global.index = module.exports; // Why not global.MyLib ?
  }
})(this, function (exports, module) {
```

**Comment by sebmck** [comment]
Fixed via 6b76f26ed899ae703b64ea5cd543641cf7b67a3d.

**Comment by sebmck** [comment]
Fixed as of 5.0.9, thanks!
